tests/testthat/test-lambda-path.R

```diff
@@ -0,0 +1,43 @@
+context("lambda path computations")
+
+test_that("lambda paths are computed appropriately", {
+  library(glmnet)
+  glmnet.control(fdev = 0)
+
+  n <- 100
+  d <- 3
+
+  # TODO(jolars): test for sparse with standardization once in place centering
+  # has been implemented
+
+  for (sparse in c(TRUE, FALSE)) {
+    x <- Matrix::rsparsematrix(n, d, density = 0.2)
```

Comment:
I like to have a `set.seed` call whenever I use random numbers in tests. Makes debugging much easier...